module_utils/network_lsr/argument_validator.py

```diff
@@ -1553,6 +1604,15 @@ def _validate_post(self, value, name, result):
                         "but is '%s'" % result["mac"],
                     )
 
+            if "match" in result:
```

Comment:
we can change `if "match" in result:` into `result.get("match")`, but we can not change  `if "path" in result["match"]:` into `result["match"].get("path")`, because when setting is defined as `"match":{"path": []}`, `if result["match"].get("path")` will evaluate to false, the remaining validation can not execute.